test/test_padacioso.py

```diff
@@ -49,15 +49,15 @@ def test_intents(self):
         })
         self.assertEqual(container.calc_intent('buy beer'), {
             'name': 'buy', 'entities': {'item': 'beer'},
-            "conf": 0.95  # unseen entity example
+            "conf": 0.9  # unseen entity example
         })
         self.assertEqual(container.calc_intent('eat some bananas'), {
             'name': 'eat', 'entities': {'fruit': 'bananas'},
-            "conf": 0.9  # unregistered entity
+            "conf": 0.96  # unregistered entity
```

Comment:
the test above should report higher conf than this one

Reply:
I disagree, the test above has an intent where valid `item` entities are defined and an invalid entity is provided; this example provides no constraints for what a `fruit` is, so "bananas" is more valid than "beer".

Reply:
"should" as in, to keep previous behavior, but i guess you may have found a scoring issue

We really should come up with a better scoring formula

Reply:
I agree; a better intent service should allow for pre-tagged entity classification requirements so we don't have to rely on lists of strings to define entities